wiegand/exceptions.py

```diff
@@ -0,0 +1,11 @@
+# Exceptions for Wiegand 26 Bit Object
+
+# When Card returns wrong Facility 
+class FacilityError(Exception):
+    pass
+
+# When Card returns wrong bit parity  
```

Comment:
when does it happen and how often does it happen? is it the hardware failure or software failure? if it's the software failure, can we handle it internally or correct it in some way instead of returning an error? same question for the facility error above.